database/models.py

```diff
@@ -10,6 +10,9 @@ class Base(DeclarativeBase):
     __abstract__ = True  # Indicates that this class should not be created as a table
     id = Column(String(36), primary_key=True, default=lambda: str(uuid.uuid4()))
 
+    def to_dict(self):
```

Comment:
that's a really nice helper method! we can probably clean up the other one in the interface in favor of this one eventually.